python/unix/unix.cmake

```diff
@@ -56,6 +57,14 @@ endif()
 
 set(PYNCPP_PYTHON_STDLIB_INSTALL_DIR "python${PYNCPP_PYTHON_VERSION_MAJOR}.${PYNCPP_PYTHON_VERSION_MINOR}")
 
+if(APPLE)
```

Comment:
if Apple ?

Reply:
it's for macos specific  code

Reply:
indeed but the executed code for apple looks like more of a windows line: `set(executable_name "python.exe")`. This could be a bug

Reply:
Python compiles the executable with the .exe extension on MacOS because the filesystem is case-insensitive (which means "Python" and "python" are equal), and there is already a `Python` folder in the build tree.

Reply:
oh ok, thx, at one point it could be explained in a comment since it's not immediately clear